### PR TITLE
Game letters class

### PR DESCRIPTION
**Pull request by Tyzlous**
Made game letters class, ready for use.

### CODE REVIEW

File: TeamHangman/TeamHangman/Game.cpp [hunk 1]
```diff
@@ -75,6 +80,44 @@ void Game::WordVectorInit()
...
+	{
+		firstLetterPos = 0.05f;
+		lastLetterPos = 0.95f - firstLetterPos;
+		letterSpacing = window->getSize().x * lastLetterPos / word.size();
+	}
+	else
+	{
+		firstLetterPos = 0.1f;
+		lastLetterPos = 0.9f - firstLetterPos;
+		letterSpacing = window->getSize().x * lastLetterPos / word.size();
```
Comment (Tachiibou) [inline TeamHangman/TeamHangman/Game.cpp]: We could calculate letterSpacing just once, below these three if, else if, else statements.

File: TeamHangman/TeamHangman/Game.cpp [hunk 2]
```diff
@@ -151,6 +198,14 @@ void Game::print(std::string string)
...
 	std::cout << letter << std::endl;
-	std::cout << GetRandomLine() << std::endl;
+	for (int i = 0; i < gameLetters.size(); i++)
+	{
+		if (gameLetters[i]->CompareToMyLetter(letter) == true)
+		{
+			correctLetters++;
+		}
+	}
+	// if (correctLetters == 0) {failedGuess();}
```
Comment (Tachiibou) [inline TeamHangman/TeamHangman/Game.cpp]: gonna use this comment later?

File: TeamHangman/TeamHangman/Game.cpp (hunk 1 above)
Comment (Tyzlous) [reply TeamHangman/TeamHangman/Game.cpp]: true dat

File: TeamHangman/TeamHangman/GameLetter.cpp [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	{
+		delete label;
+	}
+	if (bottomLine != nullptr)
+	{
+		delete bottomLine;
+	}
+}
+
+void GameLetter::Update()
```
Comment (Tachiibou) [inline TeamHangman/TeamHangman/GameLetter.cpp]: Are we going to use this empty Update function?

File: TeamHangman/TeamHangman/Game.cpp (hunk 2 above)
Comment (Tyzlous) [reply TeamHangman/TeamHangman/Game.cpp]: do not have the hangman yet to make anything happen

File: TeamHangman/TeamHangman/GameLetter.cpp (hunk 3 above)
Comment (Tyzlous) [reply TeamHangman/TeamHangman/GameLetter.cpp]: probably not

**Comment by Tachiibou** [comment]
I tink i fixed merge conflicts

**Pull request merged**